File: requests.kt
    import khttp.*

    fun main() {
        val url = "http://example.com/api/data"

        try {
            val response = khttp.get(url)

            if (response.statusCode == 200) {
                println(response.text)
            } else {
                println("Request failed with status code ${response.statusCode}")
            }
        } catch (e: Exception) {
            println("Error: ${e.message}")
        }

        try {
            val response = khttp.put(url)

            if (response.statusCode == 200) {
                println(response.text)
            } else {
                println("Request failed with status code ${response.statusCode}")
            }
        } catch (e: Exception) {
            println("Error: ${e.message}")
        }

        try {
            val response = khttp.post(url)

            if (response.statusCode == 200) {
                println(response.text)
            } else {
                println("Request failed with status code ${response.statusCode}")
            }
        } catch (e: Exception) {
            println("Error: ${e.message}")
        }

        try {
            val response = khttp.head(url)

            if (response.statusCode == 200) {
                println(response.text)
            } else {
                println("Request failed with status code ${response.statusCode}")
            }
        } catch (e: Exception) {
            println("Error: ${e.message}")
        }
    }

File: requests.py
import requests

#url = 'http://example.com/api/data'
response = requests.get('http://example.com/api/data')

if response.status_code == 200:
    print(response.text)
else:
    print(f'Request failed with status code {response.status_code}')


response = requests.post('http://example.com/api/data')

if response.status_code == 200:
    print(response.text)
else:
    print(f'Request failed with status code {response.status_code}')

response = requests.put('http://example.com/api/data')

if response.status_code == 200:
    print(response.text)
else:
    print(f'Request failed with status code {response.status_code}')    